set.mm

```diff
@@ -507408,6 +507711,276 @@ such that every prime ideal contains a prime element (this is a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Polynomial quotient and polynomial remainder
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    r1padd1.p $e |- P = ( Poly1 ` R ) $.
+    r1padd1.u $e |- U = ( Base ` P ) $.
+    r1padd1.n $e |- N = ( Unic1p ` R ) $.
+    ${
+      q1pdir.d $e |- ./ = ( quot1p ` R ) $.
+      q1pdir.r $e |- ( ph -> R e. Ring ) $.
+      q1pdir.a $e |- ( ph -> A e. U ) $.
+      q1pdir.c $e |- ( ph -> C e. N ) $.
+      ${
+        q1pdir.b $e |- ( ph -> B e. U ) $.
+        q1pdir.1 $e |- .+ = ( +g ` P ) $.
+        $( Distribution of univariate polynomial quotient over addition.
+           (Contributed by Thierry Arnoux, 2-Apr-2025.) $)
+        q1pdir $p |- ( ph -> ( ( A .+ B ) ./ C ) = ( ( A ./ C ) .+ ( B ./ C ) )
+          ) $=
+          ( wcel crg co cmulr cfv csg cdg1 clt wbr ply1ring syl ringgrpd grpcld
+          wceq q1pcl syl3anc uc1pcl eqid ringdir syl13anc cabl ringabld ringcld
+          oveq2d ablsub4 syl122anc eqtrd fveq2d cr1p syl2anc r1pcl eqeltrrd cxr
+          r1pval deg1xrcl r1pdeglt eqbrtrrd deg1addlt eqbrtrd w3a q1peqb biimpa
+          wa syl32anc ) AHUATZBCGUBZITZDJTZBDEUBZCDEUBZGUBZITZWEWJDFUCUDZUBZFUE
+          UDZUBZHUFUDZUDZDWPUDZUGUHZWEDEUBWJUMZOAIGFBCLSAFAWDFUATZOFHKUIUJZUKZP
+          RULQAIGFWHWILSXCAWDBITZWGWHITZOPQIJFEHBDNKLMUNUOZAWDCITZWGWIITZORQIJF
+          EHCDNKLMUNUOZULAWQBWHDWLUBZWNUBZCWIDWLUBZWNUBZGUBZWPUDWRUGAWOXNWPAWOW
+          EXJXLGUBZWNUBZXNAWMXOWEWNAXAXEXHDITZWMXOUMXBXFXIAWGXQQIJFHDKLMUPUJZIG
+          FWLWHWIDLSWLUQZURUSVCAFUTTXDXGXJITXLITXPXNUMAFXBVAPRAIFWLWHDLXSXBXFXR
+          VBAIFWLWIDLXSXBXIXRVBIGFWNXLBCXJLSWNUQZVDVEVFVGAIWPGHXKXMWRFKWPUQZOLS
+          ABDHVHUDZUBZXKIAXDXQYCXKUMPXRIFEHWLYBBDWNYBUQZKLNXSXTVMVIZAWDXDWGYCIT
+          OPQIJFHYBBDYDKLMVJUOVKACDYBUBZXMIAXGXQYFXMUMRXRIFEHWLYBCDWNYDKLNXSXTV
+          MVIZAWDXGWGYFITORQIJFHYBCDYDKLMVJUOVKAXQWRVLTXRIWPFHDYAKLVNUJAYCWPUDZ
+          XKWPUDWRUGAYCXKWPYEVGAWDXDWGYHWRUGUHOPQIJWPFHYBBDYDKLMYAVOUOVPAYFWPUD
+          ZXMWPUDWRUGAYFXMWPYGVGAWDXGWGYIWRUGUHORQIJWPFHYBCDYDKLMYAVOUOVPVQVRWD
+          WFWGVSWKWSWBWTIJWPFEHWLWEDWNWJNKLYAXTXSMVTWAWC $.
+      $}
+
+      ${
+        q1pvsca.1 $e |- .X. = ( .s ` P ) $.
+        q1pvsca.k $e |- K = ( Base ` R ) $.
+        q1pvsca.8 $e |- ( ph -> B e. K ) $.
+        $( Scalar multiplication property of the polynomial remainder
+           operation.  (Contributed by Thierry Arnoux, 2-Apr-2025.) $)
+        q1pvsca $p |- ( ph -> ( ( B .X. A ) ./ C ) = ( B .X. ( A ./ C ) ) ) $=
```

Comment:
```suggestion
        $( Scalar multiplication property of polynomial division.
           (Contributed by Thierry Arnoux, 2-Apr-2025.) $)
        q1pvsca $p |- ( ph -> ( ( B .X. A ) ./ C ) = ( B .X. ( A ./ C ) ) ) $=
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4746/commits/c42e7d79eba289c072cab906104237b2826cbc6a